scripts/run_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
I am not sure that I want to add Bash to this project... Is there any way that this could be in xonsh?